hotsos/core/host_helpers/apparmor.py

```diff
@@ -0,0 +1,78 @@
+from functools import cached_property
+
+# NOTE: we import direct from searchkit rather than hotsos.core.search to
+#       avoid circular dependency issues.
+from searchkit import (
+    FileSearcher,
+    SearchDef,
+    SequenceSearchDef,
+)
+from hotsos.core.factory import FactoryBase
+from hotsos.core.host_helpers.cli import CLIHelperFile
+
+
+class AAProfile(object):
+
+    def __init__(self, name):
+        self.name = name
+        self.mode = ApparmorHelper().get_profile_mode(name)
```

Comment:
There is a 'processes' section in apparmor_status should that be added?

Reply:
i dont have a current need for it (was adding to help with https://github.com/canonical/hotsos/pull/724) but we can always extend later if needs be.